LLama/LLamaExecutorBase.cs

```diff
@@ -262,7 +262,7 @@ protected virtual void TryReuseMatchingPrefix()
         /// <param name="inferenceParams"></param>
         /// <param name="args"></param>
         /// <returns></returns>
-        protected abstract Task<(bool, IReadOnlyList<string>)> PostProcess(IInferenceParams inferenceParams, InferStateArgs args);
```

Comment:
This is a breaking change for anything inheriting from executor base. So it needs to stay async.